ingestion/functions/retrieval/retrieval.py

```diff
@@ -1,8 +1,11 @@
 import codecs
+import io
 import json
+import mimetypes
 import os
 import sys
-import io
+import tempfile
+import zipfile
```

Comment:
Sanity checking -- these are all python3 and don't require declaration in `requirements.txt`, right?

Reply:
yup, python has "battery included" as they like to say